src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,32 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User user = new User();
+        user.setEmail("user@server.com");
+        user.setPassword("password");
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+        try {
+            User registerUser = authenticationService.register(user.getEmail(), user.getPassword());
+            System.out.println(registerUser);
+
+        } catch (RegistrationException e) {
+            System.out.println(e.getMessage());
+        }
+        try {
+            User loginUser = authenticationService.login(user.getEmail(), user.getPassword());
+            System.out.println(loginUser);
+        } catch (AuthenticationException e) {
+            System.out.println(e.getMessage());
```

Comment:
it would be better to `throw new RuntimeException(myCustomMessage, e)`

Reply:
ok, but it's only test code